### ISSUE TITLE
search for kanji by English meaning not returning all results

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nelenkov/wwwjdic#47]
```
Looks like kanji whose translation matches exactly the search key are not 
returned, just ones that include it in the translation string. The WWWJDIC web 
interface seems to handle this correctly.

```

Original issue reported on code.google.com by `nikolay....@gmail.com` on 3 Jun 2011 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
Was using the wrong backdoor code. Need to use 'M' for Unicode search key.
```

Original comment by `nikolay....@gmail.com` on 5 Jun 2011 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
fixed in 1.8.3
```

Original comment by `nikolay....@gmail.com` on 6 Jun 2011 at 1:41
- Changed state: **Fixed**
